### ISSUE TITLE
Locations management

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/psulib_blacklight#1182]
Locations are currently being used to populate the location dropdown in the advanced search, and the location display in the availability.  We are able to pull location codes and descriptions from Symphony, however, the descriptions are abbreviated or sometimes incorrect and the data otherwise will need to be curated by and admin.

We would like to have a single place to store locations from Symphony and allow an admin to edit them.

Potential solution:

- Create data model called `Locations` with `code`, `symphony_description`, and `preferred_description` attributes
- Create a script to import Symphony codes and descriptions to regularly import this data
- Create a basic CRUD interface behind admin authentication that displays the Symphony code and description and allows the user to update the preferred description
- Alter availability and advanced search location logic to pull from this data

This solution stems from these issues: https://github.com/psu-libraries/psulib_traject/issues/267 https://github.com/psu-libraries/psulib_blacklight/issues/425

**Comment by ajkiessl** [comment]
Also, here's a script in traject for pulling locations from Symphony: https://github.com/psu-libraries/psulib_traject/pull/470/files .  That code can be deleted once this issue is complete.

**Comment by Smullz622** [comment]
on hold for #1591
